lib/battery-percentage.js

```diff
@@ -0,0 +1,26 @@
+var events = require('events');
+var util = require('util');
+
+var five = require('johnny-five');
+
+function BatteryPercentage(opts) {
+  this._sensor = new five.Sensor({
+    pin: 'BATP',
+    range: [0x00, 0xff]
```

Comment:
I haven't had time to run this on my CHIP yet, but is the range of 0 - 255 correct? What range of values does the `BAT_PERCENT_REGISTER` contain?